jquery-mockjax/jquery-mockjax.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for jQuery Mockjax 2.0.1
+// Project: https://github.com/jakerella/jquery-mockjax
+// Definitions by: Laszlo Jakab <https://github.com/laszlojakab/>, Vladimir Đokić <https://github.com/vladeck/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../jquery/jquery.d.ts" />
+
+interface MockJaxSettings {
+    url?: string | RegExp;
+    data?: any;
+    type?: string;
+    headers?: any;
+    logging?: boolean;
+    status?: number;
+    statusText?: string;
+    responseTime?: number;
+    isTimeout?: boolean;
+    contentType?: string;
+    response?: (settings: any) => void;
```

Comment:
That is a function, just defined as having one argument that can be of any type and returning void. If we define it just as Function, then it would be unclear as to the "interface" of that function (what does it return, what are it's arguments).